### PR TITLE
LPAL-422 Plug in feedback endpoint to flask-front, spin up in docker and test 

### PR DESCRIPTION
**Pull request by nickdavis2001**
- Plug the feedback endpoint into the Flask site provided by opg-flask-apps repo
- Spin up this site in the docker-compose stack with a healthcheck in CI
- Tests to ensure the feedback endpoint works and double-check branding is still OK

### CODE REVIEW

File: .github/workflows/build_images.yml [hunk 1]
```diff
@@ -50,13 +49,15 @@ jobs:
 
       - name: Build and test front docker image
         run: |
-          pip install wait-for-it
+          pip install wait-for-it pytest pytest-cov
```
Comment (townxelliot) [inline .github/workflows/build_images.yml]: Does this repeat what `pip install -e .` does later (line 59)?

File: .github/workflows/build_images.yml [hunk 2]
```diff
@@ -50,13 +49,15 @@ jobs:
...
           which wait-for-it
```
Comment (townxelliot) [inline .github/workflows/build_images.yml:53]: Redundant line?

File: feedbackfront/opgfeedbackfront/feedbackfront_blueprint.py [hunk 3]
```diff
@@ -0,0 +1,11 @@
+from flask import Blueprint, render_template
+from flask import jsonify
+from flask import current_app
+import logging
```
Comment (townxelliot) [inline feedbackfront/opgfeedbackfront/feedbackfront_blueprint.py]: I think only line 1 is importing anything useful.

**Review by townxelliot — CHANGES_REQUESTED** [review]

File: feedbackfront/opgfeedbackfront/feedbackfront_blueprint.py (hunk 3 above)
Comment (nickdavis2001) [reply feedbackfront/opgfeedbackfront/feedbackfront_blueprint.py]: Yep these were old imports from before. Now removed.

File: .github/workflows/build_images.yml (hunk 2 above)
Comment (nickdavis2001) [reply .github/workflows/build_images.yml:53]: Removed

File: .github/workflows/build_images.yml (hunk 1 above)
Comment (nickdavis2001) [reply .github/workflows/build_images.yml]: I wasn't previously aware that dev dependencies got pulled in by default. Makes sense now so have removed this duplicate install, and at the same time done the same for the feedbackapi. (added dev deps in setup.py and removed duplicate pytest install)

**Review by townxelliot — DISMISSED** [review]

**Review by townxelliot — APPROVED** [review]

**Pull request merged**